decision-policy.php

```diff
@@ -14,27 +14,37 @@
 		<p><a href="https://www.w3.org/2018/Process-20180201/#Consensus">Consensus</a> is not a vote. The exact number of working group participants supporting a Call for Consensus compared to objections is not the only factor in the decision. Although significant support from the active membership is always desirable, consensus means working through objections until they are resolved either through amending the decision or in rare cases overriding the objection as laid out in <a href="https://www.w3.org/2018/Process-20180201/#managing-dissent">Managing Dissent</a>. In order to work through objections they must have a clear rationale based on the technical merit or with reference to the agreed scope of the work. Moving on usually means a careful approach is taken. For example, <em>not</em> adding something to the documentation.</p>
 		<p>Additions to normative text such as new success criteria should have a pre-defined scope. For example, the <a href="https://www.w3.org/WAI/GL/wiki/WCAG_2.1_Success_Criteria">requirements for WCAG 2.1 success criteria</a>. Where new normative text does not reach consensus the reasons should be recorded, depending on the origin of the text. For example, if the new text originated in a github issue or pull request, it could be labeled with "Unsupported addition".</p>
-		<p>During discussion on a topic, participants are welcome to raise objections freely to help ensure that all available information can be considered and contribute to the best possible decision. However, when the chairs issue a Call for Consensus, objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise on points that the individual considers suboptimal but can "live with" is an essential part of group decisions that must meet various requirements. </p>
+		<p>During discussion on a topic, participants are welcome to raise objections freely to help ensure that all available information can be considered and contribute to the best possible decision. However, when the chairs issue a Call for Consensus (CfC), objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise on points that the individual considers suboptimal but can "live with" is an essential part of group decisions that must meet various requirements.  Objections from people who raised their concerns to the group during discussions may be given more consideration than objections from people who had opportunity and chose to not participate in those discussions.</p>
```

Comment:
This makes it sound like the objections are not evaluated as much on their merits but on whether they have been raised unsuccessfully before.

Reply:
Suggest the following replace the whole paragraph from the second sentence onward (change of "when" to "once" in first sentence; many changes in subsequent):
However, once the chairs issue a Call for Consensus (CfC), objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise is an essential part of group decisions. This requires individuals to participate actively in modes of discussion, but "live with" points they consider suboptimal once reaching CfC.